components/molecule/pagination/src/index.js

```diff
@@ -83,6 +92,7 @@ const MoleculePagination = ({
             onClick={handleClickPrev}
             design={prevButtonDesign}
             disabled={!prevPage}
+            {...linkProps(prevPage)}
```

Comment:
If component does not receive linkFactory as a prop, it will break

Reply:
it won't, `linkProps` checks whether `linkFactory` is a function